ignite/services/chain/init.go

```diff
@@ -235,3 +233,71 @@ type Account struct {
 	CoinType string
 	Coins    string
 }
+
+func createValidatorFromConfig(conf *v1.Config) Validator {
+	// Currently, we support the config file with one valid validator.
+	validator := conf.ListValidators()[0]
+	name := validator.Name
+	monikerV := ""
```

Comment:
Do we need to define all these vars? We can directly mutate the Validator struct.